Automattic/newspack-blocks
# fix(ras-acc): add product summary card to auth flow

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1206943664367857/1207328906416559/f

This PR adds the product price summary card to the auth flow:

![Screenshot 2024-05-27 at 23 32 41](https://github.com/Automattic/newspack-blocks/assets/17905991/eb4b14c6-bdb9-4776-896b-8f2bb5603756)

### How to test the changes in this Pull Request:

Before testing, be sure to checkout the `epic/ras-acc` branch in the main Newspack Plugin.

- Add a donation block to any page or post.
- Add some checkout buttons the same page or post. Be sure to include:
  - A simple product
  - A simple subscription
  - A variable product
  - A variable subscription
- As a logged out reader, go to the page or post where the above have been added
- Click the donate button and confirm the auth modal opens with a product summary card at the top of the modal with the product name and price (as pictured above).
- Repeat the above steps with other donation options, as well as with each of the checkout buttons

### Other information:
